test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -483,6 +484,94 @@ public void PackCommand_SymbolPackageWithNuspecFile()
             }
         }
 
+        [Fact]
+        public void PackCommand_SymbolPackageWithNuspecFileAndReference()
```

Comment:
It would be good to express the expected outcome in the test name:

    <MethodUnderTest>_<Conditions>_<Expectation>

For example:

    PackCommand_WhenPackingSymbolsPackage_ExcludesAssemblyReferences